controllers/util/secrets_util.go

```diff
@@ -53,6 +53,9 @@ type TargetSecretFormat string
 const (
 	SecretFormatRamen  TargetSecretFormat = "ramen"
 	SecretFormatVelero TargetSecretFormat = "velero"
+
+	// This is a dev time assertion message to detect any new unhandled format in related functions
+	unknownFormat = "detected unhandled target secret format"
```

Comment:
(nit) Should the `u` be capital? I don't know if that is intentional.

Reply:
Ack to what @nirs stated, kept it private as use was within the module and did not need to be exported.

Reply:
True. In that case, we may need to just remove it and return `fmt.Error` from where it is being used.